extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/caffeine/CaffeineCacheInfo.java

```diff
@@ -17,6 +17,8 @@ public class CaffeineCacheInfo {
 
     public boolean metricsEnabled;
 
+    public boolean cacheNullValues = true;
```

Comment:
The default value will come from the configuration.
```suggestion
    public boolean cacheNullValues;
```